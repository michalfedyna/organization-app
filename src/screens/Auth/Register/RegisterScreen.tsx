import React from 'react';
import {Container} from '@components/Layout';

const RegisterScreen = () => {
  return <Container />;
};

export default RegisterScreen;
