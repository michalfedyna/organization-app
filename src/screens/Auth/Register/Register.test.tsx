import React from 'react';
import {test, expect} from '@jest/globals';
import renderer from 'react-test-renderer';

import RegisterScreen from './RegisterScreen';

test('RegisterScreen renders correctly', () => {
  const tree = renderer.create(<RegisterScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
