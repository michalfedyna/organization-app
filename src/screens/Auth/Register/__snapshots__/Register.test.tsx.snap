// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterScreen renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
/>
`;
